e2e/commands/patientOperations.ts

```diff
@@ -14,6 +14,7 @@ export interface Patient {
     dead: boolean;
     deathDate?: any;
     causeOfDeath?: any;
+    preferredAddress: any;
```

Comment:
Let's define the type for the address. @ayush-AI 

```suggestion
    preferredAddress: {
        address1: string,
        // Fill other attributes
    };
```

Reply:
Sure, updated the PR.